### PR TITLE
[NO GBP] Donation Item for Vexcint

### PR DESCRIPTION
**Pull request by sqnztb**
## About The Pull Request

Adds the Anubite Headpiece as a donator item. 

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

</details>

## Changelog

:cl:
add: Adds Vexcint's donation item.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 1]
```diff
@@ -1725,3 +1725,12 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/poster/contraband/korpstech, 32)
...
 
 #undef NOBILITY_DRESSCOAT_WASHING_CREDITS_NEEDED
+
+/obj/item/clothing/head/anubite
+	name = "Anubite Headpiece"
+	desc = "A dark coloured headpiece with golden accents. It's features seem reminiscent of the god Anubis."
+	icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/head.dmi'
+	icon_state = "anubite_headpiece"
+	worn_icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/hats.dmi'
+	worn_icon_state = "anubite_headpiece"
```
Comment (ghost) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]:
```suggestion
```
It shouldn't _need_ this second define, "icon_state" will by default apply to the worn_icon_state...
Hopefully. Some base items break it. If clothing/head does then... welp.

Comment (sqnztb) [reply modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]: I swear when I originally tested it, I got an empty sprite when wearing it. Seems to work fine now though so.

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 2]
```diff
@@ -1736,3 +1736,11 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/poster/contraband/korpstech, 32)
 	can_adjust = FALSE
 
 #undef NOBILITY_DRESSCOAT_WASHING_CREDITS_NEEDED
+
+/obj/item/clothing/head/anubite
+	name = "Anubite Headpiece"
+	desc = "A dark coloured headpiece with golden accents. It's features seem reminiscent of the god Anubis."
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]:
```suggestion
	desc = "A dark coloured headpiece with golden accents. Its features seem reminiscent of the god Anubis."
```

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 3]
```diff
@@ -725,6 +725,11 @@
 	item_path = /obj/item/clothing/suit/toggle/labcoat/medical/vic_dresscoat_donator
 	ckeywhitelist = list("nikotheguydude")
 
+/datum/loadout_item/suit/anubite_headpiece
+	name = "Anubite headpiece"
```
Comment (tf-4) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]:
```suggestion
	name = "Anubite Headpiece"
```

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 4]
```diff
@@ -1736,3 +1736,11 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/poster/contraband/korpstech, 32)
 	can_adjust = FALSE
 
 #undef NOBILITY_DRESSCOAT_WASHING_CREDITS_NEEDED
+
+/obj/item/clothing/head/anubite
+	name = "Anubite Headpiece"
```
Comment (ghost) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]: I WANT to say this should be all lowercase _but_ I think Anubite would be a proper noun in this case... ughhhhhhnbhbhgbhg it likely doesn't matter

Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]:
```suggestion
	name = "\improper Anubite headpiece"
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Review by GoldenAlpharex — APPROVED** [review]
Should be good now, good work!

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 5]
```diff
@@ -1736,3 +1736,11 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/poster/contraband/korpstech, 32)
 	can_adjust = FALSE
 
 #undef NOBILITY_DRESSCOAT_WASHING_CREDITS_NEEDED
+
+/obj/item/clothing/head/anubite
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm:1741]:
```suggestion
//  Donation reward for vexcint
/obj/item/clothing/head/anubite
```

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**